contrast_security/user_api/application_api.py

```diff
@@ -128,11 +128,12 @@ def filter_applications(self, org_uuid, application_filter=None):
         if application_filter is None:
             application_filter = ApplicationFilter()
         path = '{org_uuid}/applications/filter'.format(org_uuid=org_uuid)
-        return self._get(path, params=application_filter.get_params_as_json())
+        return self._post(path, json=application_filter.get_body_params_as_json(),
+                          params=application_filter.get_query_params_as_json())
 
     def get_application_filters(self, org_uuid):
         path = '{org_uuid}/applications/filters/listing'.format(org_uuid=org_uuid)
-        return self._get(path)
+        return self._post(path)
```

Comment:
this method should be a GET
see contrast.teamserver.rest.ng.application.NgApplicationsRestController.getApplicationsAdvancedFilters(Organization)

Reply:
Yes, you're right. I take the original changes as guide and after that I run the correspond test and showed not problem. seems because the lack of body and query params the method call is not failing. anyway I'm returning to get this method.